packages/editor/src/panels/inspector/inspectoreditor.tsx

```diff
@@ -0,0 +1,101 @@
+/*
+CPAL-1.0 License
+
+The contents of this file are subject to the Common Public Attribution License
+Version 1.0. (the "License"); you may not use this file except in compliance
+with the License. You may obtain a copy of the License at
+https://github.com/ir-engine/ir-engine/blob/dev/LICENSE.
+The License is based on the Mozilla Public License Version 1.1, but Sections 14
+and 15 have been added to cover use of software over a computer network and 
+provide for limited attribution for the Original Developer. In addition, 
+Exhibit A has been modified to be consistent with Exhibit B.
+
+Software distributed under the License is distributed on an "AS IS" basis,
+WITHOUT WARRANTY OF ANY KIND, either express or implied. See the License for the
+specific language governing rights and limitations under the License.
+
+The Original Code is Infinite Reality Engine.
+
+The Original Developer is the Initial Developer. The Initial Developer of the
+Original Code is the Infinite Reality Engine team.
+
+All portions of the code written by the Infinite Reality Engine team are Copyright © 2021-2023 
+Infinite Reality Engine. All Rights Reserved.
+*/
+
+import { toDisplayDateTime } from '@ir-engine/common/src/utils/datetime-sql'
+import { getMutableState, useHookstate } from '@ir-engine/hyperflux'
+import Text from '@ir-engine/ui/src/primitives/tailwind/Text'
+import React from 'react'
+import { useTranslation } from 'react-i18next'
+import { FiEdit2 } from 'react-icons/fi'
+import { ClickPlacementState } from '../../systems/ClickPlacementSystem'
+
+const InspectorEditor = () => {
+  const { t } = useTranslation()
+  const { metadata } = useHookstate(getMutableState(ClickPlacementState)).value
+  const { thumbnail, name, type, author, dateCreated, fileSize, dimensions, mesh, resources, tags } = metadata
+  const leftTableCellCss = `table-cell w-2/5 text-left font-bold py-0.5`
+  const rightTableCellCss = `table-cell text-left py-0.5`
+
+  return Object.keys(metadata).length > 0 ? (
+    <div className="flex h-full flex-col gap-6 overflow-y-auto bg-surface-3 p-3 px-10 text-text-secondary">
+      <div className="align-center flex justify-center">
+        <img src={thumbnail} alt={name} className="m-3 text-center" />
+      </div>
+      <div className="flex flex-row items-center justify-between">
```

Comment:
This could use a slight `gap-x` to create space between the text and icon

![image](https://github.com/user-attachments/assets/b5c99edb-381f-4bcf-b29c-3dfd826d1fb1)